silkrpc/core/blocks.cpp

```diff
@@ -47,10 +52,44 @@ boost::asio::awaitable<uint64_t> get_highest_block_number(const core::rawdb::Dat
 }
 
 boost::asio::awaitable<uint64_t> get_latest_block_number(const core::rawdb::DatabaseReader& reader) {
+    const auto kv_pair = co_await reader.get(silkrpc::db::table::kLastForkchoice, silkworm::bytes_of_string(std::string("headBlockHash")));
```

Comment:
I suggest the following changes to improve readability:
- `silkrpc::db::table` -> `db::table`
- `silkworm::bytes_of_string(std::string("headBlockHash"))` -> `silkworm::bytes_of_string("headBlockHash")`
- define constant for "headBlockHash" at namespace level